src/routes/jira/jira-get.ts

```diff
@@ -187,7 +187,7 @@ const renderJiraCloudAndEnterpriseServer = async (res: Response, req: Request):
 
 	const hasConnections =  !!(installations.total || gheServers?.length);
 
-	await saveConfiguredAppProperties(jiraHost, undefined, undefined, req, hasConnections ? "true" : "false");
+	await saveConfiguredAppProperties(jiraHost, undefined, undefined, req.log, hasConnections ? true : false);
```

Comment:
/nit Its already boolean. 
```suggestion
	await saveConfiguredAppProperties(jiraHost, undefined, undefined, req.log, hasConnections);
```